composer.json

```diff
@@ -9,16 +9,18 @@
             "type": "composer",
             "url": "https://packages.drupal.org/8"
         },
+        "1": {
+            "type": "composer",
+            "url": "https://asset-packagist.org"
+        },
         "km_collaborative": {
             "type": "git",
             "url": "https://github.com/United-Philanthropy-Forum/km-collaborative"
         }
 
     },
     "require": {
-        "php": ">=7.0.8",
-        "pantheon-systems/example-drops-8-composer": "^2.3",
-        "united-philanthropy-forum/km_collaborative": "*"
+        "united-philanthropy-forum/km_collaborative": "^0.1.4"
```

Comment:
This means >=0.1.4,<1.0.0 -- we probably want to avoid any kind of upper limit, right? Like this:

`"united-philanthropy-forum/km_collaborative": ">=0.1.4"`

I had to look this up, because I always do: https://getcomposer.org/doc/articles/versions.md#caret-version-range-

Reply:
I love it